var/spack/repos/builtin/packages/ftgl/package.py

```diff
@@ -48,6 +51,13 @@ class Ftgl(AutotoolsPackage):
     depends_on('glu')
     depends_on('freetype@2.0.9:')
 
+    # Currently, "make install" will fail if the docs weren't built
+    #
+    # FIXME: Can someone with autotools experience fix the build system
+    #        so that it doesn't fail when that happens?
+    #
+    depends_on('doxygen')
```

Comment:
This should probably be `type='build'`

Reply:
You are right! Fixed in the latest push.